test/toInteger32.js

```diff
@@ -0,0 +1,74 @@
+import * as R from 'ramda';
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('toInteger32', function () {
+  context('given a small integer', function () {
+    specify('should maintain the same value', function () {
+      assert.strictEqual(RA.toInteger32(1 + Math.random()), 1);
+      assert.strictEqual(RA.toInt32(1 + Math.random()), 1);
+    });
+
+    specify('should preserve the sign', function () {
+      assert.strictEqual(RA.toInteger32(-1 - Math.random()), -1);
+      assert.strictEqual(RA.toInt32(-1 - Math.random()), -1);
+    });
+  });
+
+  context('given an integer 1 modulo 23^2', function () {
+    specify('should give one when converted', function () {
+      const initialValue = 2 ** 32 + 1;
+      assert.strictEqual(RA.toInteger32(initialValue), 1);
+      assert.strictEqual(RA.toInt32(initialValue), 1);
+    });
+  });
+
+  context('given an integer -1 modulo 23^2', function () {
+    specify('should give one when converted', function () {
+      const initialValue = 2 ** 32 - 1;
+      assert.strictEqual(RA.toInteger32(initialValue), -1);
+      assert.strictEqual(RA.toInt32(initialValue), -1);
+    });
+  });
+
+  context('given invalid input', function () {
+    specify('should return 0', function () {
+      assert.strictEqual(RA.toInteger32(false), 0);
+      assert.strictEqual(RA.toInt32(false), 0);
+
+      assert.strictEqual(RA.toInteger32([]), 0);
+      assert.strictEqual(RA.toInt32([]), 0);
+
+      assert.strictEqual(RA.toInteger32(null), 0);
+      assert.strictEqual(RA.toInt32(null), 0);
+
+      assert.strictEqual(RA.toInteger32(NaN), 0);
+      assert.strictEqual(RA.toInt32(NaN), 0);
+
+      assert.strictEqual(RA.toInteger32(/a/), 0);
+      assert.strictEqual(RA.toInt32(/a/), 0);
+
+      assert.strictEqual(RA.toInteger32({}), 0);
+      assert.strictEqual(RA.toInt32({}), 0);
+
+      assert.strictEqual(RA.toInteger32(new Error()), 0);
+      assert.strictEqual(RA.toInt32(new Error()), 0);
+    });
+  });
+
+  context('given Infinity value', function () {
+    specify('should return 0', function () {
+      assert.strictEqual(RA.toInteger32(Infinity), 0);
+      assert.strictEqual(RA.toInt32(Infinity), 0);
```

Comment:
Let's also add tests for `-Infinity`.
```suggestion
      assert.strictEqual(RA.toInt32(Infinity), 0);
      assert.strictEqual(RA.toInt32(-Infinity), 0);
      assert.strictEqual(RA.toInt32(-Infinity), 0);      
```

Reply:
👍